node/src/Pos/Wallet/Web/ClientTypes/Types.hs

```diff
@@ -110,6 +112,13 @@ newtype CPassPhrase = CPassPhrase Text
 instance Show CPassPhrase where
     show _ = "<pass phrase>"
 
+-- | CBOR-encoded data.
+newtype CEncodedData = CEncodedData BSL.ByteString
+    deriving (Eq, Generic)
+
+instance Buildable CEncodedData where
+  build _ = "<encoded data>"
```

Comment:
Minor detail: Maybe it would be better for the logging to log the string with the encoded data?

Reply:
:+1:

Reply:
Fixed in #8.